src/main/java/hw10/FileLoggerConfiguration.java

```diff
@@ -0,0 +1,54 @@
+package hw10;
+
+import java.io.File;
+import java.io.IOException;
+
+public class FileLoggerConfiguration {
+    private File file;
+    private static LoggingLevel loggingLevel;
+    private long maxFileSize;
+    private String messageFormat;
+
+    public static void loadTextToFile(String message) throws IOException {
```

Comment:
Sorry, but this is not correct. I would suggest removing this method.
Your configuration should have only the specific parameters and the ability to fill/access them, nothing more.